vars/skipStage.groovy

```diff
@@ -200,7 +248,9 @@ boolean call(Map config = [:]) {
             return skip_stage_pragma('vagrant-test', 'true') &&
                    ! env.BRANCH_NAME.startsWith('weekly-testing')
         case "Coverity on CentOS 7":
-            return skip_stage_pragma('coverity-test') ||
+            return params_value('CI_BUILD_PACKAGES_ONLY', false) ||
+                   params.CI_RPM_TEST_VERSION ||
```

Comment:
If this is going to be a skip condition, shouldn't `rpmTestVersion() != ''` also be added?

Reply:
I did not realize that was not already covered.

Reply:
Ultimately, can coverity be used when doing a run from RPMs built from a previous run?  If not, then this should be added in addition to your parameter addition.  If it can, then your parameter check here shouldn't be here.

Reply:
Coverity is actually a quickbuild build stage that will build the git commit for the Jenkins job if it is allowed to run.

So any time that you are skipping a normal build, or specifying a pre-built RPM, a coverity build is not wanted or is not going to build what was expected.

The `rpmTestversion() != ''` includes a test for param.RPM_TEST_VERSION so both are not needed in the same conditional and will be removed in the next commit.